swarm/api/content_type.go

```diff
@@ -0,0 +1,57 @@
+package api
```

Comment:
i don't think that there's justification for a new file here. split the relevant parts into `api.go` and `server.go` (see below)